wiki/telescope_design.md

```diff
@@ -125,9 +127,17 @@ To get started, we need to make sure that all our wires are hooked up and ready
 
 ![pi setup](assets/rasp_set_up.jpeg){:height="50%" width="50%"}
 
+The CHART Pi image comes with all the software you need to run CHART and analyze data.
```

Comment:
Lets be more specific than "run CHART".  How about "The CHART Pi image comes pre setup ready to observe the 21cm line and analyze the data."